proposals/low-level-struct-improvements.md

```diff
@@ -442,7 +443,9 @@ The [rationale](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-
 
 <a name="rules-unscoped"></a>
 
-To fix this the  language will provide the opposite of the `scoped` lifetime annotation in the syntax `unscoped`.  The keyword `unscoped` will be used to expand the lifetime of a value. It can be applied to any `ref` which is implicitly `unscoped` and has the impact of changing its *ref-safe-to-escape* to *calling method*.
+To fix this the  language will provide the opposite of the `scoped` lifetime annotation by supporting an `UnscopedRefAttribute`.
+The attribute can be used to expand the lifetime of a value. It can be applied to any `ref` which is [implicitly `scoped`](#implicitly-scoped) and has the impact of changing its *ref-safe-to-escape* to *calling method*.
+An error is reported if `[UnscopedRef]` is applied to a parameter passed by value or to a `ref` that is not considered scoped.
```

Comment:
>not considered scoped. [](http://example.com/codeflow?start=104&length=22)

"not considered implicitly scoped"? That will cover the `[UnscopedRef] scoped` scenario as well (line 500). Can this line (448) be moved to line 500 for various errors reported on misused `[UnscopedRef]`?